scripts/marumaru/parse.ts

```diff
@@ -69,12 +70,16 @@ export function parse(html: string, sourceUrl: string) {
     const translation = el.text().trim()
     const line = lyrics.find(line => line.pk === pk)
     if (line) {
+      const localeCode = 'zh-Hant'
```

Comment:
We don't need to change this. This is the script for the crwaler of jpmarumaru.com, which is always zh-Hant